sql/src/main/java/io/crate/analyze/TableIdentsExtractor.java

```diff
@@ -162,13 +161,8 @@
         }
 
         @Override
-        public Collection<RelationName> visitQueriedTable(QueriedTable table, Void context) {
-            return Collections.singletonList(table.tableRelation.tableInfo().ident());
-        }
-
-        @Override
-        public Collection<RelationName> visitQueriedDocTable(QueriedDocTable table, Void context) {
-            return process(table.tableRelation(), context);
+        public Collection<RelationName> visitQueriedTable(QueriedTable<?> queriedTable, Void context) {
+            return Collections.singletonList(queriedTable.tableRelation().tableInfo().ident());
```

Comment:
as we are already have table relation visit method, imho it would make sense to use them here as well by `process(queriedTable.tableRelation(), context);`